fix minimum time lag stripping, add prec to min time lag

rerpha opened the pull request.

rerpha committed:
fix minimum time lag stripping, add prec to min time lag

Tom-Willemsen commented on correlator_pcaspy.py:
```diff
@@ -276,7 +276,9 @@ def take_data(self) -> None:
         wait_in_seconds = self.get_converted_pv_value(Records.WAIT.name)
         wait_at_start = self.get_converted_pv_value(Records.WAIT_AT_START.name)
-        min_time_lag = self.get_converted_pv_value(Records.MIN_TIME_LAG.name)
+        min_time_lag_unscaled = self.get_converted_pv_value(Records.MIN_TIME_LAG.name)
```
Would it be clearer to call this `min_time_lag_ns` or something like that?

Tom-Willemsen commented on correlator_pcaspy.py:
```diff
@@ -276,7 +276,9 @@ def take_data(self) -> None:
-        min_time_lag = self.get_converted_pv_value(Records.MIN_TIME_LAG.name)
+        min_time_lag_unscaled = self.get_converted_pv_value(Records.MIN_TIME_LAG.name)
+        # Convert min time lag to seconds for comparison against lag data
+        min_time_lag = min_time_lag_unscaled / 1e9
```
magic number

```suggestion
        min_time_lag = min_time_lag_unscaled / NANOSECONDS_TO_SECONDS
```

Tom-Willemsen reviewed (changes requested):
- This causes the existing IOC tests to fail
- Please add IOC tests covering the cases which are fixed by this PR.

rerpha committed:
pull out magic number, rename min_time_lag_unscaled to min_time_lag_ns

rerpha commented:
> * This causes the existing IOC tests to fail
> * Please add IOC tests covering the cases which are fixed by this PR.

thanks for the review, have addressed comments and added [this PR](https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/587) - i think the tests fixed cover the issue so didn't feel i needed to create another one
